src/services/analytics/events/overview.ts

```diff
@@ -117,6 +117,12 @@ export const OVERVIEW_EVENTS = {
     category: OVERVIEW_CATEGORY,
     //label: OPEN_SAFE_LABELS
   },
+  // Track clicks on links to Safe Accounts
+  EXPAND_MULTI_SAFE: {
+    action: 'Expand multi Safe',
+    category: OVERVIEW_CATEGORY,
+    //label: OPEN_SAFE_LABELS
```

Comment:
should the label be included?

Reply:
Its just a comment that should signal that its an even which uses one of OPEN_SAFE_LABELS as label. (Copied it from a related event)